Cargo.toml

```diff
@@ -12,24 +12,28 @@ benchmark = []
 async-fn = { path = "crates/async-fn" }
 async-std = { version = "=1.6.0", features = ["unstable"] }
 async-trait = "0.1.36"
+bytes = "0.5.6"
 console_log = { version = "0.2" }
 console_error_panic_hook = { version = "0.1.1", optional = true }
 data-encoding = "1.1.1"
 flatbuffers = "0.6.1"
 futures = "0.3.5"
 http = "0.2.1"
+# hyper implies tokio below. Should be dead-code-removed in wasm.
+hyper = "0.13"
```

Comment:
> It's weird that it doesn't pull it in itself? Is that because the runtime it uses is meant to be pluggable?

We depend directly on it here https://github.com/rocicorp/repc/pull/80/files#diff-0de8ae7a07d52b7410d27f30e00a32dbR75

> I'm a bit concerned about pulling in tokio ... because it seems like it overlaps with functionality provided by async-std, which we've already chosen. Will that cause problems?

I don't know.

I didn't think we were even using the async_std runtime but it looks like I was mistaken: https://github.com/rocicorp/repc/blob/d4f67ab80ddb33b7556055c14ec7be4e56d1e2fc/src/kv/idbstore.rs#L3. I'm a little unsure about that because I'm not sure it's possible to prevent it from spawning threads without explicit configuration, and it has some magic to "detect blocking" (look for that phrase in https://async.rs/blog/stop-worrying-about-blocking-the-new-async-std-runtime/). I don't see us setting ASYNC_STD_THREAD_COUNT anywhere. I think @whiten said in slack he is looking at this as a potential cause of a problem, possibly related to the next thing I think.

We are primarily using the wasm_bindgen runtime via https://rustwasm.github.io/wasm-bindgen/api/wasm_bindgen_futures/fn.spawn_local.html. This should work both in wasm and in regular rust I think. 

Regarding tokio's runtime, I *thought* that we we weren't using it because we are not using the attribute "#[tokio::main]" or anything else that obviously configures or uses the runtime like tokio's task::spawn. I assumed that we were just creating the http request future and then when we await it it's all running on our main thread. However I looked under the hood into the http client and indeed it is using a tokio executor and tokio tasks. I have no idea what kinds of problems to expect if we are both spawning tasks with wasm_bindgen and with tokio. I know that in principle you can plug other runtimes into tokio and have it work, but no idea how hard that would be in this case. However the good news is that this potential problem is not present in wasm, just in not-wasm and and thus potentially in integration tests.

I think we should spin the runtime question out into an issue.

Reply:
https://github.com/rocicorp/repc/issues/82